packages/ui-kit/src/Toggler/style.module.css

```diff
@@ -57,6 +57,23 @@
   }
 }
 
+.twoLabels {
+  .toggle {
+    background-color: var(--base-strong);
+    margin: 0 0.5em 0 0.5em;
+  }
+  .label {
+    font-weight: 600;
```

Comment:
I think it would be better to keep `font-size`, `font-weight` and `line-height` the same by default, regardless of whether it's regular or two-directional toggle. If we need to make the text bigger in some specific place, we should provide className for the labels